src/main/resources/org/sagebionetworks/web/client/widget/accessrequirements/ACTAccessRequirementWidgetViewImpl.ui.xml

```diff
@@ -61,6 +61,14 @@
 					and Compliance Team (ACT) to discuss at act@sagebase.org
         </bh:Text>
       </bh:Div>
+      <bh:Div styleName="margin-top-10">
+        <g:InlineLabel styleName="boldText" text="Entities Covered" />
+        <bh:Hr addStyleNames="margin-top-5 margin-bottom-5" />
+      </bh:Div>
+      <bh:Div
+        ui:field="accessRequirementRelatedProjectsListContainer"
```

Comment:
`swc.css` has list styles that override the MUI list styles. Add a class name to the div so that styles can be re-specified.